### DISCUSSION TITLE
SVGEdit in React and TypeScript application

### DISCUSSION THREAD

**Discussion by mike2003** [SVG-Edit/svgedit#926] [General]
Any chance of running this inside a React and TypeScript application? I've been trying all day, but none of the instructions work.

**Comment by jfhenon** [comment]
Hello,
You have a sample React application at https://github.com/SVG-Edit/svg-edit-react
It was not updated recently but it should be working and can certainly be used as a starter.
Thanks
JF

**Comment by manufac-pbansal** [comment]
How can I run this inside my typescript application? Seems like no type definitions are available for `svgedit`. Any solution?
